Ryujinx.HLE/HOS/Services/Am/AppletAE/AllSystemAppletProxiesService/SystemAppletProxy/ISelfController.cs

```diff
@@ -298,6 +298,15 @@ public ResultCode ReportUserIsActive(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandHipc(67)] //3.0.0+
+        //IsIlluminanceAvailable() -> bool
+        public ResultCode IsIlluminanceAvailable(ServiceCtx context)
+        {
+            Logger.Stub?.PrintStub(LogClass.ServiceAm);
+            context.ResponseData.Write(true); // This should call through to Lbl, but there's no situation where we'd want false.
```

Comment:
```suggestion
            // NOTE: This should call through to Lbl, but there's no situation where we'd want false.
            context.ResponseData.Write(true);
            
            Logger.Stub?.PrintStub(LogClass.ServiceAm);
```
Small formatting nit.